File: store/src/main/java/com/nytimes/android/external/store2/util/ParserException.java
package com.nytimes.android.external.store2.util;

/**
 * Exception thrown when one of the provided parsers fails.
 */
public class ParserException extends RuntimeException {

    public ParserException(Throwable cause) {
        super(cause);
    }

    public ParserException(String message) {
        super(message);
    }

    public ParserException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: store/src/main/java/com/nytimes/android/external/store3/util/ParserException.kt
package com.nytimes.android.external.store3.util

/**
 * Exception thrown when one of the provided parsers fails.
 */
class ParserException : RuntimeException {

    constructor(cause: Throwable) : super(cause)

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)
}
